### PR TITLE
added notes about OnPrem_Distinguished_Name

### PR DESCRIPTION
**Pull request by MaratMussabekov**
https://github.com/MicrosoftDocs/windows-itpro-docs/issues/9438

### CODE REVIEW

**Comment by MaratMussabekov** [comment]
Dear @JohanFreelancer9,
kindly do the copy editor review for this PR.

Dear @nam31, for your information

Thank you

File: windows/security/identity-protection/hello-for-business/hello-hybrid-aadj-sso-cert.md [hunk 1]
```diff
@@ -679,6 +679,11 @@ Sign-in a workstation with access equivalent to a _domain user_.
 
 10. Select **Enroll to Windows Hello for Business, otherwise fail (Windows 10 and later)** from the **Key storage provider (KSP)** list.
 11. Next to **Subject name format**, type **CN={{OnPrem_Distinguished_Name}}** to make the on-premises distinguished name the subject of the issued certificate.
+
+   > [!Note]
```
Comment (illfated) [inline windows/security/identity-protection/hello-for-business/hello-hybrid-aadj-sso-cert.md:683]:
```suggestion
   > [!NOTE]
```

File: windows/security/identity-protection/hello-for-business/hello-hybrid-aadj-sso-cert.md [hunk 2]
```diff
@@ -679,6 +679,11 @@ Sign-in a workstation with access equivalent to a _domain user_.
...
+   > If the distinguished names contain scpecial characters ("+", ",", ";" or "="), put quotation marks: CN=”{{OnPrem_Distinguished_Name}}”.
```
Comment (illfated) [inline windows/security/identity-protection/hello-for-business/hello-hybrid-aadj-sso-cert.md:684]:
```suggestion
   > If the distinguished name contains special characters ("+", ",", ";" or "="), the bracketed name must be enclosed in quotation marks: CN=”{{OnPrem_Distinguished_Name}}”.
```

File: windows/security/identity-protection/hello-for-business/hello-hybrid-aadj-sso-cert.md [hunk 3]
```diff
@@ -679,6 +679,11 @@ Sign-in a workstation with access equivalent to a _domain user_.
...
+   > If the distinguished names length is more than 64 characters, name length enforcement on the Certification Authority [must be disabled](https://docs.microsoft.com/previous-versions/windows/it-pro/windows-server-2003/cc784789(v=ws.10)?#disable-dn-length-enforcement)
```
Comment (illfated) [inline windows/security/identity-protection/hello-for-business/hello-hybrid-aadj-sso-cert.md:685]:
```suggestion
   > If the length of the distinguished name is more than 64 characters, the name length enforcement on the Certification Authority [must be disabled](https://docs.microsoft.com/previous-versions/windows/it-pro/windows-server-2003/cc784789(v=ws.10)?#disable-dn-length-enforcement).
```

**Comment by illfated** [comment]
The added note text might be phrased differently from what I have suggested. If you happen to know how exactly how it needs to be written to get the information across correctly, please recommend further improvements to my suggestions above.

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - only @illfated's suggestions.

**Comment by illfated** [comment]
@MaratMussabekov : Looks like my suggestions are good to use, please commit the suggestions.

**Review by mapalko — APPROVED** [review]

**Review by illfated — APPROVED** [review]
Looks good to me.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by illfated** [comment]
#sign-off

**Comment by e0i** [comment]
@tiburd This PR has been approved by the author and can be merged. FYI.

File: windows/security/identity-protection/hello-for-business/hello-hybrid-aadj-sso-cert.md [hunk 4]
```diff
@@ -679,6 +679,11 @@ Sign-in a workstation with access equivalent to a _domain user_.
 
 10. Select **Enroll to Windows Hello for Business, otherwise fail (Windows 10 and later)** from the **Key storage provider (KSP)** list.
 11. Next to **Subject name format**, type **CN={{OnPrem_Distinguished_Name}}** to make the on-premises distinguished name the subject of the issued certificate.
+
+   > [!NOTE]
+   > If the distinguished name contains special characters like plus sign ("+"), comma (","), semicolon (";"), or equal sign ("="), the bracketed name must be enclosed in quotation marks: CN=”{{OnPrem_Distinguished_Name}}”.
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/hello-for-business/hello-hybrid-aadj-sso-cert.md:684]:
```suggestion
   > If the distinguished name contains special characters like a plus sign ("+"), comma (","), semicolon (";"), or equal sign ("="), the bracketed name must be enclosed in quotation marks: CN=”{{OnPrem_Distinguished_Name}}”.
```

File: windows/security/identity-protection/hello-for-business/hello-hybrid-aadj-sso-cert.md [hunk 5]
```diff
@@ -679,6 +679,11 @@ Sign-in a workstation with access equivalent to a _domain user_.
...
+   > If the length of the distinguished name is more than 64 characters, the name length enforcement on the Certification Authority [must be disabled](https://docs.microsoft.com/previous-versions/windows/it-pro/windows-server-2003/cc784789(v=ws.10)?#disable-dn-length-enforcement).
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/hello-for-business/hello-hybrid-aadj-sso-cert.md:685]:
```suggestion
   > If the length of the distinguished name is more than 64 characters, the name length enforcement on the Certification Authority [must be disabled](/previous-versions/windows/it-pro/windows-server-2003/cc784789(v=ws.10)?#disable-dn-length-enforcement).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
@MaratMussabekov, I have approved this PR quite a while back, before the links rule change. Could you kindly also commit these two suggested changes? Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]
Thank you @MaratMussabekov.

**Pull request merged**